### ISSUE TITLE
Further development despite "Appointment" option in Nextcloud calendar?

### ISSUE THREAD

**Issue by CodeAdminDe** [SergeyMosin/Appointments#358]
Hey, I found this app and would like to use it long term (and then support it accordingly). 

However, I wonder: Will the app be developed in parallel to the Nextcloud Appointments (calendar-integrated) in the long term?

This app offers a few basic features that I desperately need that the built-in calendar can't (privacy checkbox, talk integration, ...).

Anway, thanks for the work!

**Comment by Flashpok** [comment]
Same question.
What are your plans ?
thanks,

**Comment by xeruf** [comment]
just wanted to ask, as this seems to be the same, I thought it might be related to this plugin?
![image](https://user-images.githubusercontent.com/13354331/191129994-11427e7a-e2f5-437c-ac12-73e4395cba67.png)

**Comment by SergeyMosin** [comment]
There are plans for version 2 which will include backed code rewrite for performance and extensibility. Additionally I want to implement a new front facing design, probably something similar to this https://github.com/SergeyMosin/Appointments/issues/157#issuecomment-731593642 . There was also a plan to add functionality to integrate with and/or extended the native appointments app.

Current version ( v1 ) will only get bug fixes.

I am somewhat busy with an other project so v2 is on the back-burner for now. However, as the first step, I'll put a user survey page up next week or so, to prioritize feature requests and better plan for v2.

**Comment by CodeAdminDe** [comment]
That sounds awesome. Please keep us in the loop. Would be happy to support the project when I get spare time again to forward my private projects.

**Comment by xeruf** [comment]
Are you planning on working with upstream to improve and build on the feature rather than creating a parallel ecosystem? Having two options will confuse users, and already does.

**Comment by chrlii** [comment]
Are there any updates for this topic?

I am using @SergeyMosin 's appointment app at the moment. And I find the second option within the calendar app confusing, too, as @xeruf already mentioned. I like and need parts of the features of each option that are missing in the other.

What I like (and need) with the appointments app is:
- User can select from different locations for the meeting.
- Possibiliy of customised fields.
- Privacy check can be added.

What I like with calendar/appointment app is:
- Look an feel of the form with calendar picker design. Photo of the inviting person etc.


This would be nice to have in the "older" appointments app, too. Any plans to add this?

For me a show stopper with the "newer" calendar/appointment app is, that privacy check box cannot be added and only one location can be offered. (If I have understood it right?) Any plans to add this here?

What are the plans for these two apps?
Are there any plans integrationg this two apps?
